### PR TITLE
[BUGFIX] Avoid double CI runs for pull requests

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # https://help.github.com/en/categories/automating-your-workflow-with-github-actions
 
 on:
-  pull_request:
   push:
+    branches:
+      - main
```
Comment (JakeQZ) [inline .github/workflows/ci.yml:6]: This should currently be `master`.  The repo hasn't had the anti-slavery rename yet.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
Rename of main branch to `main` still needs to be done, unless I've missed that ticket.

**Comment by JakeQZ** [comment]
cf https://github.com/MyIntervals/emogrifier/pull/1251

File: .github/workflows/ci.yml (hunk 1 above)
Comment (oliverklee) [reply .github/workflows/ci.yml]: Done.

**Review by JakeQZ — APPROVED** [review]

**Pull request merged**